tests/php/test-amp-helper-functions.php

```diff
@@ -1745,23 +1755,28 @@ public function test_amp_add_admin_bar_item() {
 
 		$this->go_to( amp_get_permalink( $post_id ) );
 
+		delete_site_transient( 'theme_roots' );
+
```

Comment:
It doesn't appear that this does anything anymore:

```suggestion
		delete_site_transient( 'theme_roots' );

```